components/splitbutton/stories/splitbutton.stories.js

```diff
@@ -16,9 +16,19 @@ export default {
       options: ["s", "m", "l", "xl"],
       control: "select"
     },
+    variant: {
+      name: "Variant",
+      type: { name: "string", required: true },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      options: ["cta", "primary", "secondary"],
+      control: "select",
+    }
```

Comment:
Does this need `iconName: { table: { disable: true } },` added?

Reply:
Yes! I can add it.